TDAT: public void test invalid data section2 ( ) throws exception {  <NL> try {  <NL>  _ capi . call data section2 ( " hello " ) ;  <NL> fail ( " expected unacceptable result xins call exception " ) ;  <NL>  } catch ( unacceptable result xins call exception exception ) {  <NL>  / / as expected <NL>  } catch ( exception exception ) {  <NL> exception . print stack trace ( ) ;  <NL> fail ( " the result is invalid , the function should throw an unacceptable result xins call exception exception " ) ;  <NL>  }  <NL>  }  COM: <s> tests that the returned attribute types are checked </s>